examples/stream-decode/README.md

```diff
@@ -0,0 +1,5 @@
+### Decode SCTE-35 from a MPEGTS File. 
+```smalltalk
```

Comment:
smalltalk?

Reply:
smalltalk has cooler syntax highlighting :)

Reply:
Haha, hadnt considered that ;)